<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>회원가입</string>
  </property>
  <widget class="QLineEdit" name="sb_nameData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sb_nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>이 름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="sb_idLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="sb_idData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sb_passLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="sb_passData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sb_queLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>비밀번호를 잊어버렸을 경우에 해당질문을 제시하여 본인임을 확인합니다. 
본인만 알 수 있는 질문과 그 답을 선택해주세요!</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>질 문</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="sb_ansLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>정 답</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="sb_ansData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="sb_queData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>192</y>
     <width>197</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>197</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentText">
    <string>질문을 선택해주세요!</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>26</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>질문을 선택해주세요!</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>어린시절 가장 오래 살았던 동네는?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>나의 전화번호 가운데 네 자리를 거꾸로 하면?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>내가 사람을 처음 볼 때 가장 중요하게 생각하는 것은?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>내가 가진 것 중에서 가장 비싼 것은?</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sb_moneyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>총 자산과 월급은 여러분들의 소득분위를 기준으로 개인에게 
맞는 데이터를 제공해주기 위해 얻는 데이터입니다.
(개인정보는 데이터 가공 외에는 사용되지 않습니다!)</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>총 자산</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="sb_moneyData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>                                 (만원)</string>
   </property>
  </widget>
  <widget class="QLabel" name="sb_payLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>총 자산과 월급은 여러분들의 소득분위를 기준으로 개인에게 
맞는 데이터를 제공해주기 위해 얻는 데이터입니다.
(개인정보는 데이터 가공 외에는 사용되지 않습니다!)</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>월 급</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="sb_payData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>                                 (만원)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sb_subscribe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid Red;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sb_back">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid Red;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>홈화면으로</string>
   </property>
  </widget>
  <widget class="QLabel" name="sb_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border: 0.9px solid Red;
padding: 2px;</string>
   </property>
   <property name="text">
    <string>회원가입 페이지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
